modules6/Assignment #5.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Assignment 5</title>
+<meta charset="utf-8">
+<meta name="viewport" content="width-device-width, initial-scale=1">
+<link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css"/>
+<link rel="stylesheet" href="_css/assign5.css"/>
+<script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+<script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+<!-- Ted Pysh
+	index.html
+	info2439
+	Thoendel
+	9/8/19 
+	-->
+
+
+</head>
+<body>
+<div data-role="page">
+	<div data-role="header" data-position="fixed" data-id="headernav" data-theme="a">
+	<h1>Theodore Pysh</h1>
+	<div data-role="header">
+	<h1>Assignment 5, Sage Bistro Static Google Map </h1>
+	</div>
+	<div role="main" class="ui-content">
+		<div class="ui-grid-a">
+			<div class="ui-block-a">
+			<img width="150" 
+			src="https://maps.googleapis.com/maps/api/staticmap?center=MCC+Sage+Student+Bistro,+Omaha,+NE&zoom=13&scale=1&size=150x150&maptype=roadmap&key=AIzaSyAFQid4GkVK5cIqrk3n8kuRkAo7YcNeTzo&format=jpg&visual_refresh=true&markers=size:mid%7Ccolor:0xff0000%7Clabel:1%7CMCC+Sage+Student+Bistro,+Omaha,+NE" 
```

Comment:
This looks like you used my API key. I would then expect some screenshots either here in your repo or in your BB submission showing that you created your own credentials without billing attached.